experimental/PropertyDDS/packages/property-changeset/src/changeset_operations/arrayChangesetIterator.ts

```diff
@@ -0,0 +1,290 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * @fileoverview Iterator to iterate over array ChangeSets
+ */
+
+// @ts-ignore
+import { constants } from "@fluid-experimental/property-common";
+import _ from "lodash";
```

Comment:
Just FYI - Wholesale import of 'lodash' has a noticeable effect on the PropertySet bundle size (#3925).

Reply:
Yes, we have a task in our backlog to enable tree shaking on lodash imports for all property packages.